src/main/java/tc/oc/pgm/commands/MapCommands.java

```diff
@@ -73,8 +73,11 @@ public static void maplist(
       throws CommandException {
     final Set<PGMMap> maps = ImmutableSortedSet.copyOf(library.getMaps());
 
-    int resultsPerPage = 8;
-    int pages = library.getMaps().size() / resultsPerPage + 1;
+    double resultsPerPage = 8;
+    int pages =
+        library.getMaps().size() / resultsPerPage <= 0
+            ? 1
+            : (int) Math.ceil(library.getMaps().size() / resultsPerPage);
 
```

Comment:
This looks unnecessary, none of the variables you're dealing with needs to be a double. 
Note that division between two integers in Java returns the resultant value with the fractional part truncated away.

Reply:
I also though it was unnecessary but it is needed so the ceiling returns the upper number else it will just return the integer value. Example: 9 / 8 as integer returns 1 but using one value as double returns 2

Reply:
All being ints: `(size + resultsPerPage - 1) / resultsPerPage`